src/main/java/io/mosip/mimoto/constant/SwaggerLiteralConstants.java

```diff
@@ -0,0 +1,77 @@
+package io.mosip.mimoto.constant;
+
+public class SwaggerLiteralConstants {
+
+    /*  Attestation Controller */
+    public static final String ATTESTATION_NAME = "Attestation" ;
+    public static final String ATTESTATION_DESCRIPTION = "All the Attestation Related Endpoints" ;
+
+
+    /*  Common Inji Controller */
+    public static final String COMMON_INJI_NAME = "Properties" ;
+    public static final String COMMON_INJI_DESCRIPTION = "All the Inji Wallet Properties Related Endpoints" ;
+    public static final String COMMON_INJI_GET_PROPERTIES_SUMMARY = "Retrieve All Inji Wallet Properties" ;
```

Comment:
Write "Retrieve all Inji Wallet Properties" instead of "Retrieve All Inji Wallet Properties"

Reply:
Updated